en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-rest-to-soap-api.md

```diff
@@ -0,0 +1,290 @@
+
+You can use the following steps to configure the **Micro Integrator** to update the APIM service catalog for the developped integrations.
+
+1. Update the metadata of the Integration Service.
+    - Open the `PhoneVerify_metadata.yaml` file located in `PhoneVerification/PhoneVerificationConfigs/src/main/resources/metadata/PhoneVerify_metadata.yaml` using the project explorer.
+    - Change the `serviceUrl` as follows.
+
+        ```url tab="Linux"
+        http://172.17.0.1:8290/phoneverify
+        ```
+
+        ```url tab="MacOS/Windows"
+        http://host.docker.internal:8290/phoneverify
+        ```
+
+        !!! Note
+            `serviceUrl` is used to access service you created on Micro Integrator. When you create the API on API Manager, this `serviceUrl` will be used as the endpoint for your service.
+            
+            Also note that this guide assumes, you are using `docker-compose` to run API Manager and Choreo Connect Gateway and **Micro Integrator** is running on the host machine(localhost). Therefore, If you are using another setup, please make sure to set the host & port of the `serviceUrl` accordingly.
+        
+2. Configure Micro integrator to automatically publish artifacts to **Service Catalog** in API Manager publisher portal.
+    - Open the server configurations for Embedded Micro Integrator in Integration Studio by clicking on the **Server Configuration(Micro Integrator)** (<a href="{{base_path}}/assets/img/integrate/tutorials/common/server-config-64x64.png"><img src="{{base_path}}/assets/img/integrate/tutorials/common/server-config-64x64.png" width="20"></a>) icon on the upper menu.
+    - Uncomment the `[[service_catalog]]` section and change the APIM server configurations accordingly.
+        ```toml
+        [[service_catalog]]
+        apim_host = "https://localhost:9444"
+        enable = true
+        username = "admin"
+        password = "admin"
+        ```
+
+### Step 4 - Update Swagger Definition
+
+This step will show you how to Update the swagger definition to change some important parameters which is required when deploy this API in Choreo Connect Gateway using the Swagger definition.
+
+#### Change Server URL
+
+!!! Note
+    Follow this step only if you are using Choreo Connect in [Standalone Mode]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-rest-to-soap-api/#via-apictl-for-standalone-mode) on `docker-compose` and **Micro Integrator** is running on `localhost`.
+
+1. Open the file in `PhoneVerification/PhoneVerificationConfigs/src/main/resources/metadata/PhoneVerify_swagger.yaml`.
+2. Change the URL under `servers` section as follows.
+
+    ```url tab="Linux"
+    http://172.17.0.1:8290/phoneverify
+    ```
+
+    ```url tab="MacOS/Windows"
+    http://host.docker.internal:8290/phoneverify
+    ```
+
+#### Update Query Parameters in Swagger definition
```

Comment:
can remove this

Reply:
fixed via https://github.com/wso2/docs-apim/pull/6114/commits/d03a84262c3c0fa449b465e58f50d312a1220157